Mathlib/Combinatorics/Matroid/Basic.lean

```diff
@@ -665,7 +665,7 @@ theorem Indep.exists_insert_of_not_isBase (hI : M.Indep I) (hI' : ¬M.IsBase I)
   obtain ⟨x, hxB', hx⟩ := exists_of_ssubset (hIB'.ssubset_of_ne (by (rintro rfl; exact hI' hB')))
   by_cases hxB : x ∈ B
   · exact ⟨x, ⟨hxB, hx⟩, hB'.indep.subset (insert_subset hxB' hIB')⟩
-  obtain ⟨e,he, hBase⟩ := hB'.exchange hB ⟨hxB',hxB⟩
+  obtain ⟨e, he, hBase⟩ := hB'.exchange hB ⟨hxB',hxB⟩
```

Comment:
Shouldn't the second list also have a space added?

Reply:
It probably should. Not sure why this was left out. Added now, though.